big_tests/run_common_test.erl

```diff
@@ -270,7 +270,8 @@ enable_preset_on_node(Node, PresetVars, HostVars) ->
     NewVars = lists:foldl(fun ({Var, Val}, Acc) ->
                               lists:keystore(Var, 1, Acc, {Var, Val})
                           end, Default, PresetVars),
-    NewCfgFile = bbmustache:render(Template, NewVars, [{key_type, atom}]),
+    Tmp = bbmustache:render(Template, NewVars, [{key_type, atom}]),
+    NewCfgFile = bbmustache:render(Tmp, NewVars, [{key_type, atom}]),
```

Comment:
It's quite confusing to render twice, so maybe leave here a comment as well, one of those `%% Render twice to replace variables in variables`